src/components/import/render-utils.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { BitbucketIcon, GitAltIcon, GithubIcon, GitlabIcon } from '@patternfly/react-icons';
+import { TFunction } from 'i18next';
+import CheIcon from './CheIcon';
+import { detectGitType } from '../helpers/stringHelpers';
+import { GitProvider } from '../../types';
+
+
+export const routeDecoratorIcon = (
+  routeURL: string,
+  radius: number,
+  t: TFunction,
+  cheEnabled?: boolean,
+  cheIconURL?: string,
+): React.ReactElement => {
+  if (cheEnabled && routeURL) {
+    return cheIconURL ? (
+      <image xlinkHref={cheIconURL} width={radius} height={radius} />
+    ) : (
+      <CheIcon style={{ fontSize: radius }} />
+    );
+  }
+  switch (detectGitType(routeURL)) {
+    case GitProvider.INVALID:
+      // Not a valid url and thus not safe to use
+      return null;
+    case GitProvider.GITHUB:
+      return <GithubIcon style={{ fontSize: radius }} title={t('devconsole~Edit source code')} />;
```

Comment:
Change prefixes here and add strings as our own

Reply:
Will fix this in [GITOPS-2291](https://issues.redhat.com/browse/GITOPS-2291).